src/modules/home/components/single-property/single-property-rating.tsx

```diff
@@ -0,0 +1,65 @@
+import { useState } from 'react';
+import Box from '@mui/material/Box';
+import Rating from '@mui/material/Rating';
+import Typography from '@mui/material/Typography';
+import TextField from '@mui/material/TextField';
+import Button from '@mui/material/Button';
+import { useCreateReview } from '../../api/create-review';
+
+interface ReviewFormProps {
+  accommodationId: string;
+}
+
+export function ReviewForm({ accommodationId }: ReviewFormProps) {
+  const [rating, setRating] = useState<number | null>(null);
+  const [comment, setComment] = useState<string>('');
+  const [error, setError] = useState<string>('');
+
+  const { mutate: createReview, status } = useCreateReview();
+  const isLoading = status === 'pending';
+  const handleSubmit = () => {
+    if (rating === null || comment.trim() === '') {
+      setError('Both rating and comment are required.');
```

Comment:
Comment should be optional.

Reply:
In our entity it already set nullable = false and have migration, should I change this?

Reply:
No, we can keep it like this then.